### ISSUE TITLE
Tasks without return value would cause panic when `Get` is called

### ISSUE THREAD

**Issue by sickyoon** [gocelery/gocelery#149]
Tasks without return value would cause panic when `Get` is called.

**Comment by kobayashi** [comment]
I think `Get` method works correctly.

in gocelery_test.go, each test case can get err from `Get` method with 2s timeout.
```
res, err := asyncResult.Get(TIMEOUT)
if err != nil {
	t.Errorf("test '%s': failed to get result for task %s: %+v", tc.name, tc.taskName, err)
	cli.StopWorker()
	continue
}
```
However, panic is caused at `releaseResultMessage` function in message.go because `v == nil`, right?

```
func releaseResultMessage(v *ResultMessage) {
	v.reset()
	resultMessagePool.Put(v)
}
```

**Comment by yoonsio** [comment]
@kobayashi you are right. I am curious about how Celery checks if the method has been executed without checking the result.

**Comment by yoonsio** [comment]
added test case: https://github.com/gocelery/gocelery/blob/ab4dc82c38ba560a068ade9e0d4d9eb81fc42729/gocelery_test.go#L98-L170
